src/afl-fuzz-stats.c

```diff
@@ -861,9 +864,13 @@ void show_stats(afl_state_t *afl) {
        " fuzzing strategy yields " bSTG bH10 bHT bH10 bH5 bHB bH bSTOP cCYA
        " path geometry " bSTG bH5 bH2 bVL "\n");
 
-  if (afl->skip_deterministic) {
+  if (unlikely(afl->custom_only)) {
 
-    strcpy(tmp, "n/a, n/a, n/a");
+    strcpy(tmp, "disabled (custom mutator only mode)");
```

Comment:
Somehow feels odd. `custom-mutator-only mode` or just `custom mutators only`?

Reply:
yeah "custom-mutator-only mode" is better